docs/content/en/docs/reference/changelog.md

```diff
@@ -1,14 +1,31 @@
----
-linkTitle: "Changelog"
-weight: 55
----
```

Comment:
I think we need the header so this shows up in the TOC
![Screen Shot 2021-11-01 at 14 55 21](https://user-images.githubusercontent.com/104113/139741014-0345bb92-fc75-4d04-8e98-5101141fe1df.png)

Not sure why it was removed

Reply:
was removed by mistake! added it back.